runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsBrotherSlainOverlay.java

```diff
@@ -83,10 +85,10 @@ public Dimension render(Graphics2D graphics)
 				.build());
 		}
 
-		float rewardPercent = client.getVar(Varbits.BARROWS_REWARD_POTENTIAL) / 10.0f;
+		float rewardPercent = client.getVar(Varbits.BARROWS_REWARD_POTENTIAL) / (10.0f + BrotherReward * .02f);
```

Comment:
This calculation does not look correct. Why would killing brothers increase the amount we divide by? As written, this would calculate the maximum potential as `1000 / 10.12` instead of `1012 / 10`.

Reply:
"For every Barrows brother that was slain, the player will receive an extra 2 points on their rewards potential, capping the maximum reward potential at 1,012." ~ https://oldschool.runescape.wiki/w/Barrows/Strategies#Medium%20level 
What we want to display is a percentage so we want (points/totalpoints) * 100%. As max total points in 1012, and we multiple by 100, that gives 10.12

Reply:
Is the varbit accurate to that description then? That is, does killing Ahrim add 98 or 100 to its value? Regardless, if changing the denominator happens in this PR, it becomes ambiguous what % should be stopped at to avoid getting bolt racks since the actual number of points (not the percentage) is what determines that.

Reply:
do you think then, it would be better to just give the raw points instead, sense that's what matters and that's what people know?

Reply:
thinking it over, giving the raw points seems best because it easily displays what is important, plays on what the community knows, and prevents a new variable from being made

Reply:
I agree. Since percentage isn't completely clear, it would likely be better to display points instead especially since the wiki references those same points for calculating reward potential.

Reply:
> Is the varbit accurate to that description then? That is, does killing Ahrim add 98 or 100 to its value?

The varbit is technically accurate, per Mod Kieren. You can gain 1000 total points, which is the sum of the combat levels of all the monsters you've killed with a cap of 656 from the brothers. Then, at item roll, your total points is added to 2 x # of brothers slain and this new number can reach 1012. 

The player is interested in the point value they will be rolling at, however, which is why the bonus points should likely be included the entire time in one way or another. My original suggestion was: 

> One could either show the Reward Potential and correct it by adding the invisible points, or show Base Reward Potential as the number returned from the API, Bonus Reward Potential (which is the 2 * brothers killed figure), and then lastly Total Reward Potential for maximum clarity.

I do somewhat worry that just including the invisible points and adding 100 points for a level 98 kill could confuse players considering the added points are supposed to match the combat level, which is why I think showing all of the information could potentially be useful. I'm not sure how cluttered that would feel in the UI though.